lib/node_modules/@stdlib/blas/ext/base/sdssum/lib/ndarray.native.js

```diff
@@ -47,10 +48,11 @@ var addon = require( './sdssum.native.js' );
 */
 function sdssum( N, x, stride, offset ) {
 	var view;
+	offset = minViewBufferIndex( N, stride, offset );
 	if ( stride < 0 ) {
 		offset += (N-1) * stride;
 	}
-	view = new Float32Array( x.buffer, x.byteOffset+(x.BYTES_PER_ELEMENT*offset), x.length-offset ); // eslint-disable-line max-len
+	view = offsetView( x, offset );
```

Comment:
This is the relevant file you want to emulate: https://github.com/stdlib-js/stdlib/blob/58c9a5053430b55c974e9ecccfc45c51c3fefc9e/lib/node_modules/%40stdlib/blas/base/sasum/lib/ndarray.native.js